mlrun/api/db/sqldb/db.py

```diff
@@ -546,13 +544,27 @@ def get_schedule(
 
     def delete_schedule(self, session: Session, project: str, name: str):
         logger.debug("Removing schedule from db", project=project, name=name)
+        self._delete_schedule_labels(session, project, name, commit=False)
         self._delete(session, Schedule, project=project, name=name)
 
     def _delete_schedules(self, session: Session, project: str):
         logger.debug("Removing schedules from db", project=project)
         for schedule in self.list_schedules(session, project=project):
             self.delete_schedule(session, project, schedule.name)
 
+    def _delete_schedule_labels(
```

Comment:
I see there's already duplication (function and artifact), let's fix it, make one function `_delete_class_labels` that handles them all